src/customElements.ts

```diff
@@ -60,6 +61,11 @@ export interface CustomElementInitializer {
 	(properties: WidgetProperties): void;
 }
 
+export enum ChildrenType {
```

Comment:
Just a question/random thought... for HNode and VNode we use symbols to tag the types, here we are using string enums.  Conscious decision or unconscious?

Reply:
It was conscious as an `enum` seemed more appropriate as part of a consumer API